pkg/kubelet/config/flags.go

```diff
@@ -90,6 +92,8 @@ func (s *ContainerRuntimeOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&s.DockershimRootDirectory, "experimental-dockershim-root-directory", s.DockershimRootDirectory, "Path to the dockershim root directory.")
 	fs.MarkHidden("experimental-dockershim-root-directory")
 	fs.BoolVar(&s.DockerDisableSharedPID, "docker-disable-shared-pid", s.DockerDisableSharedPID, "The Container Runtime Interface (CRI) defaults to using a shared PID namespace for containers in a pod when running with Docker 1.13.1 or higher. Setting this flag reverts to the previous behavior of isolated PID namespaces. This ability will be removed in a future Kubernetes release.")
+	fs.Int64Var(&s.DockerPIDsLimit, "docker-pids-limit", s.DockerPIDsLimit, "Tune container pids limit for docker (set -1 for unlimited), kernel >= 4.3")
```

Comment:
We shouldn't be adding anymore docker-specific flags at this point. The flag should just be `--container-pids-limit` or `--pod-pids-limit`, and should be resolved in the Kubelet.

Reply:
@tallclair - right, was my initial instinct too, but i am not sure a similar feature is available in rkt etc. hence using `docker` in the name.